src/main/java/seedu/address/logic/commands/FilterCommand.java

```diff
@@ -0,0 +1,47 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_SPORT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.client.TagAndSportContainsKeywordsPredicate;
+
+/**
+ * @author tohkerwei
+ * Filters and lists all clients in FitBiz whose tag or sports contains argument keywords for the respective parameters.
+ * Keyword matching is case insensitive.
+ */
+public class FilterCommand extends Command {
+
+    public static final String COMMAND_WORD = "filter-c";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Filters and display all clients whose tags or sports "
+            + "contains any of the specified keywords for respective parameter (case-insensitive) \n"
+            + "Parameters: t/KEYWORD... s/KEYWORD...\n"
+            + "Example: " + COMMAND_WORD + " " + PREFIX_TAG + "vegetarian overweight handsome "
+            + PREFIX_SPORT + "soccer dance basketball";
```

Comment:
Might want to give an example query where our app will actually return a few clients to the user.